components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,136 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
+            <div className="flex items-center bg-gray-50 dark:bg-gray-800 p-3 justify-between rounded-lg">
+                <div className="flex items-center overflow-hidden" title={props.image}>
+                    <span className="w-5 h-5 mr-1">
+                        <Stack />
+                    </span>
+                    <span className="truncate">{parseDockerImage(props.image).repository}</span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-500">
+                        {parseDockerImage(props.image).tag}
+                    </span>
+                </div>
+                <Button
+                    disabled={props.disabled}
+                    htmlType="button"
+                    type="transparent"
+                    className="text-blue-500"
+                    onClick={props.onClick}
+                >
+                    Change
+                </Button>
+            </div>
+        </InputField>
+    );
+}
+
+interface OrgDefaultWorkspaceImageModalProps {
+    globalDefaultImage: string | undefined;
+    settings: OrganizationSettings | undefined;
+    onClose: () => void;
+}
+
+function OrgDefaultWorkspaceImageModal(props: OrgDefaultWorkspaceImageModalProps) {
+    const [errorMsg, setErrorMsg] = useState("");
+    const [defaultWorkspaceImage, setDefaultWorkspaceImage] = useState(props.settings?.defaultWorkspaceImage ?? "");
+    const updateTeamSettings = useUpdateOrgSettingsMutation();
+
+    const handleUpdateTeamSettings = useCallback(
+        async (newSettings: Partial<OrganizationSettings>) => {
+            try {
+                await updateTeamSettings.mutateAsync({
+                    ...props.settings,
+                    ...newSettings,
+                });
+                props.onClose();
+            } catch (error) {
+                console.error(error);
+                setErrorMsg(error.message);
+            }
+        },
+        [updateTeamSettings, props],
+    );
+
+    return (
+        <Modal
+            visible
+            closeable
+            onClose={props.onClose}
+            onSubmit={() => handleUpdateTeamSettings({ defaultWorkspaceImage })}
+        >
+            <ModalHeader>Workspace Default Image</ModalHeader>
+            <ModalBody>
+                <Alert type="warning" className="mb-2">
+                    Warning: You are setting a default image for all workspaces within the organization.
+                </Alert>
+                {errorMsg.length > 0 && (
+                    <Alert type="error" className="mb-2">
+                        {errorMsg}
+                    </Alert>
+                )}
+                <div className="mt-4">
+                    <TextInputField
+                        label="Default Image"
+                        hint="Use any official or custom workspaces images from Docker Hub or any private container registries this Gitpod instance can access."
```

Comment:
**nitpick**: Minor typo and a rephrase suggestion. Does it make sense? What do you think?

```suggestion
                        hint="Use any official or custom workspace image from Docker Hub, or any from private container registries that the instance can access."
```